src/domains/collections.ts

```diff
@@ -151,6 +149,23 @@ const extractAssets = (
   );
 };
 
+const extractLicense = (collection: any) => {
+  // See https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#license
+  // license *should* be a SPDX string see https://spdx.org/licenses/ to be valid for STAC
+  const spdxLicenseRx = /^[\w\-\.\+]+$/gi;
+
+  let licenseLink = {
+    rel: "license",
+    href: "https://science.nasa.gov/earth-science/earth-science-data/data-information-policy",
+    title: "EOSDIS Data Use Policy",
+    type: "text/html",
+  };
+  const license = "proprietary";
+
+  // TODO this misses valid licenses
+  return { license, licenseLink };
```

Comment:
License and useConstraint data is all over the place in CMR. It is not standardized, this needs additional work for how to display this properly. License information is required for STAC responses, and the majority will fall under the proprietary license description, according to STAC.